server.js

```diff
@@ -210,10 +211,41 @@ module.exports.initExpress = function() {
         logger.info(`localRootFilesDir: Mapping ${config.localRootFilesDir} into /`);
         app.use(express.static(config.localRootFilesDir));
     }
+
+    // To allow for more aggressive caching of static files served from public/,
+    // we introduce a new `assets/` path that includes a cachebuster in the path.
```

Comment:
This comment won't always be so "new" 😄
```suggestion
    // we use the `assets/` path that includes a cachebuster in the path.
```